internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/shares.go

```diff
@@ -73,6 +76,8 @@ func (h *Handler) Init(c *config.Config) error {
 	h.sharePrefix = c.SharePrefix
 	h.homeNamespace = c.HomeNamespace
 
+	h.additionalInfoTemplate, _ = template.New("additionalInfo").Parse(c.AdditionalInfoAttribute)
```

Comment:
Are you not checking the error here because the only way it could fail is if the template has already been parsed? Otherwise, I think there is some value in error checking here, at least for debugging purposes.

Reply:
@refs I thought that we shouldn't fatal if the parsing fails. Similarly, in case executing the template fails, I didn't report an error but returned an empty string. I'll add a warning there